src/Microsoft.Azure.WebJobs.Logging/Entities/InstanceTableEntity.cs

```diff
@@ -112,17 +116,22 @@ internal static string RowKeyFunctionInstanceId(Guid functionInstanceId)
             return JsonConvert.DeserializeObject<IDictionary<string, string>>(this.ArgumentsJson);
         }
 
-        public string RawStatus { get; set; }
         public Guid? ParentId { get; private set; }
 
-        private FunctionInstanceStatus GetStatus()
+        Guid IFunctionInstanceBaseEntry.FunctionInstanceId
+        {
+            get
+            {
+                return Guid.Parse(this.RowKey);
```

Comment:
Yes. We're doing the same check on line 53. 

---
In reply to: [91566148](https://github.com/Azure/azure-webjobs-sdk/pull/928#discussion_r91566148) [](ancestors = 91566148)